src/components/new-request/search.vue

```diff
@@ -406,23 +412,28 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
   @Action setRequestAction!: ActionBindingIF
   @Action startAnalyzeName!: ActionBindingIF
 
+  // Constants
+  readonly colinLink = sessionStorage.getItem('CORPORATE_ONLINE_URL')
+  readonly corpOnlineLink = 'https://www.corporateonline.gov.bc.ca/'
+  readonly EntityTypes = EntityTypes
+  readonly federalAmalgamationFormLink = 'https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-' +
+    'development/business-management/permits-licences-and-registration/registries-packages/' +
+    'pack_34_xco_-_amalgamation_application_and_business_number_request.pdf'
+  readonly fullReinstatementFormLink = 'https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-' +
+    'development/business-management/permits-licences-and-registration/registries-forms/' +
+    'form_31_xco_-_full_reinstatement_application.pdf'
+  readonly limitedReinstatementFormLink = 'https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-' +
+    'development/business-management/permits-licences-and-registration/registries-forms/' +
+    'form_29_xco_-_limited_reinstatement_application.pdf'
+
   // Local properties
   corpNumValid = true
   request = null as RequestActionsI
-  jurisdiction = null
-  selectedCompanyType = null as CompanyType
-  readonly colinLink = sessionStorage.getItem('CORPORATE_ONLINE_URL')
-  readonly corpOnlineLink = 'https://www.corporateonline.gov.bc.ca/'
-  readonly EntityType = EntityType
+  jurisdiction = null as any
```

Comment:
Why are we casting this as `any` Sev? Does it matter?

Reply:
It's totally redundant and it doesn't matter.

I'm just being explicit that `jurisdiction` is untyped. Ideally we'd use a type.